src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -524,6 +524,24 @@ public function hasDatabase($name);
     public function dropDatabase($name);
 
     /**
+     * Creates the specified schema or throws an exception
+     * if there is no support for it.
+     *
+     * @param  string $schemaName Schema Name
+     * @return void
+     */
+    public function createSchema($schemaName = 'public');
```

Comment:
Though this change would break existing custom adapters, would Phinx project allows us to do such changes?